_posts/2020-02-15-pmemkv-bindings.md

```diff
@@ -0,0 +1,155 @@
+[pmemkv's README file][pmemkv_bindings_readme].
+
+### Common for bindings
+
+There are few common characteristics regarding all bindings:
+
+#### Work with libpmemkv 1.0
+
+All of our currently released bindings work properly with libpmemkv 1.0. Unfortunately their API
+may not be fully functionally equal to the native C/C++ API. These **bindings, which are
+in version 1.0** deliver API in accordance with **pmemkv 1.0**. In general, pmemkv's binding
+released in version X.Y is (always) functionally greater or equal to pmemkv in the same X.Y version.
+Additionally, if a binding's version is prior to 1.0 (any 0.x version), it should be considered
+an experimental implementation.
+
+Taking into consideration that libpmemkv is developing in the fast pace, there are always
+some gaps in the current versions of bindings' implementations. Even more, each binding
+is developed in its own pace, so the best way to check compatibility with pmemkv
+is to read changelog of the latest release of a specific binding.
+
+#### Handling pmemkv statuses
+
+Every language supports handling statuses returned from pmemkv's API. Each has its own way,
+either translating each status into numeric value or wrapping them into exceptions. Bindings
+using exceptions do not use status "OK", just returns nothing or a numeric value
+(of `PMEMKV_STATUS_OK`) if no error occurred. Return value differs also for various functions -
+to see details read binding's documentation or pmemkv's man pages.
+
+#### Not always direct access to data
+
+**Except in Python** (and soon [in Node.js][nodejs_pr]), our bindings do not (yet)
+provide direct access to the data stored in the Persistent Memory.
+They need to use buffer to copy the data to.
+
+#### Examples, tests and CI
+
+Each repository is equipped with basic, needed features like:
+* examples, showing basic library usage,
+* tests, verifying current implementation,
+* Continuous Integration system, to check all incoming changes.
+
+#### Packaging specific for language
+
+Each language is different, so we try our best to use packaging and installation systems
+native to (or one of the most common for) each binding's language.
+
+#### Contributions are welcome
+
+As all of pmem's organization repositories, the ones of pmemkv-bindings are also
+open source and in each of them any support is more than welcome.
+
+### pmemkv-nodejs
+
+Node.js binding is the first one to be released in stable version (1.0). It means
+it guarantees backward compatibility and it provides API functionally equal to
+the native pmemkv C/C++ API **in version 1.0**.
+
+It also delivers API description in form of JSDoc documentation (accessible e.g. as html in
+[here](nodejs_pmem_io)) and NodeJS-native support for configuring engines by JSON object.
+It handles returned statuses as exceptions and does not return any value when no errors occur.
+Installation of this project is supported by simply typing `npm install` within the source directory.
```

Comment:
... uses npm for distribution, and can be installed using with `npm install pmemkv-python`.